cyberbrain-vsc/src/trace_graph.js

```diff
@@ -194,7 +194,12 @@ class TraceGraph {
       ctx.textAlign = "center";
       ctx.fillStyle = "#cfa138";
       ctx.fillText(
-        `${this.traceData.frameMetadata.filename} ${this.traceData.frameMetadata.frame_name}`,
+        `${this.traceData.frameMetadata.frame_name}`,
+        0,
+        topNodePos.y - 45
+      );
+      ctx.fillText(
+        `${this.traceData.frameMetadata.filename} line ${this.traceData.frameMetadata.fn_lineno}`,
```

Comment:
Can you move filename to be above of frame name?

Reply:
Like this?
![image](https://user-images.githubusercontent.com/17478849/117344549-7d348c80-ae73-11eb-80be-c3a14226b959.png)
Feels a bit weird with the frame name squished in the middle. I was thinking frame name is most important cause that's the function you're tracing, and then the location second (filename and lineno)

Reply:
I meant something like:
```
Gits/aco2020/viz.py line 19
main2
```

Reply:
Sure